platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -61,10 +61,10 @@ private static String version(String module) {
 
 	static Stream<String> moduleDirectoryNames() throws IOException {
 		// @formatter:off
-		return Files.walk(Paths.get(".."), 1)
-				.filter(Files::isDirectory)
-				.map(Path::getFileName)
-				.map(Object::toString)
+		String startOfModuleLine = "include '";
+		return Files.readAllLines(Paths.get("../settings.gradle")).stream()
```

Comment:
Cool. I personally prefer to be explicit, but I'm sure it doesn't matter much in this case.